CHANGES_NEXT_RELEASE

```diff
@@ -23,3 +23,4 @@
 - Add: change/add path /entities/{id}/attrs (#2028)
 - Deprecate: "location" metadata to specify entity location
 - Fix: JSON response in log management operations
+- Fix: invalid URLs at subscription time that make Orion to crash (#2092)
```

Comment:
The test is not very descriptive ...

Fix: capturing invalid URLs at subscription time (and returning error response) that otherwise would make Orion crash